lib/metasploit/framework/data_service/proxy/host_data_proxy.rb

```diff
@@ -24,18 +24,27 @@ def report_host(opts)
     begin
       data_service = self.get_data_service()
       data_service.report_host(opts)
-    rescue  Exception => e
-        puts"Call to  #{data_service.class}#report_host threw exception: #{e.message}"
-        opts.each do |key, value| puts "#{key} : #{value}" end
+    rescue Exception => e
+      puts "Call to #{data_service.class}#report_host threw exception: #{e.message}"
+      opts.each do |key, value| puts "#{key} : #{value}" end
```

Comment:
This is nitpicky, but style guidelines say to use `{ |k,v| <code> }` for 1 liner blocks. I know it wasn't your code originally, but might be worth fixing.

Reply:
I actually made such a change and other simple style changes throughout the modules I touched, but reverted them to try and minimize the change set. I'm less concerned with this module than `lib/msf/ui/console/command_dispatcher/db.rb` so I'll make the correction.